downstream/modules/platform/ref-controller-automation-analytics.adoc

```diff
@@ -14,7 +14,7 @@ No credential secrets, personal data, automation variables, or task output is ga
 
 For more information, see xref:ref-controller-data-collection-details[Details of data collection].
 
-To enable this feature, turn on data collection for {Analytics} and enter your Red Hat customer credentials in the *Miscellaneous System settings* of the System configuration list of options in the {MenuAEAdminSettings} menu.
+To enable or disable this feature, use the *Gather data for {Analytics} and enter your Red Hat customer credentials in the *Miscellaneous System settings* of the System configuration list of options in the {MenuAEAdminSettings} menu.
 
```

Comment:
Suggest one of the foll. revisions:

- 'use the' -> 'use...', OR
- use the *Gather data for {Analytics} and enter your Red Hat customer credentials in the *Miscellaneous System settings* **option from ** the System configuration list.